dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/src/org/apache/cxf/ws/policy/PolicyEngineImpl.java

```diff
@@ -677,6 +686,9 @@ public boolean supportsAlternative(Collection<? extends PolicyComponent> alterna
                             }
                             return true;
                         } else {
+                            if (doLog) {
+                               LOG.fine("ignoreUnsupportedPolicy is not set, returning false 1");
+			    }
```

Comment:
Liberty change flag?

Reply:
Not needed here as this is inside start/end change block; line 693 -> } // Liberty Change End